agents/chess-agent.json

```
@@ -0,0 +1,56 @@
+{
+  "protocolVersion": "0.3.0",
+  "author": "Telex",
+  "wellKnownURI": "https://system-integration.telex.im/chessagent/.well-known/agent.json",
+  "name": "Chess Agent",
+  "description": "An agent that plays chess. Accepts moves in standard notation and returns updated board state as FEN and an image.",
+  "url": "https://system-integration.telex.im/chessagent",
+  "version": "1.0.0",
+  "provider": {
+    "organization": "Telex",
+    "url": "https://www.telex.im"
+  },
+  "documentationUrl": "https://system-integration.telex.im/chessagent/docs",
+  "capabilities": {
+    "streaming": false,
+    "pushNotifications": false,
+    "stateTransitionHistory": true
+  },
+  "authentication": {
+    "schemes": [
+      "Bearer"
+    ],
+    "credentials": null
+  },
```

The `authentication` field is not part of the official A2A Protocol specification. According to the [official A2A Protocol v0.3.0 schema](https://github.com/a2aproject/A2A/blob/main/specification/json/a2a.json), authentication should be declared using `securitySchemes` and `security` fields (following OpenAPI 3.0 specification).

Since this agent appears to be public and doesn't actually require authentication (credentials is null), you can simply remove this field. If authentication is actually required, please use the standard fields instead:

```suggestion
  },
```